src/Avalonia.Themes.Default/Controls/NativeMenuBar.xaml

```diff
@@ -4,6 +4,7 @@
        Selector="NativeMenuBar">
   <Style.Resources>
     <local:InverseBooleanValueConverter x:Key="AvaloniaThemesDefaultNativeMenuBarInverseBooleanValueConverter" Default="True"/>
+    <local:IBitmapToImageConverter x:Key="AvaloniaThemesDefaultNativeMenuBarIBitmapToImageConverterr"/>
```

Comment:
AvaloniaThemesDefaultNativeMenuBarIBitmapToImageConvert**er**

Reply:
Thanks. But already fixed in [https://github.com/AvaloniaUI/Avalonia/pull/8055/commits/6a8eb5a1cf551503c2090e3a0b8dfdc4eec8497c](url)

Reply:
> AvaloniaThemesDefaultNativeMenuBarIBitmapToImageConvert**er**

Sorry you are wright. Type still exists.
Will fix this with my next pull request for "isVisible" of NativeMenuItem